libs/wingc/src/jsify.rs

```diff
@@ -371,6 +371,13 @@ impl<'a> JSifier<'a> {
 					BinaryOperator::Sub => "-",
 					BinaryOperator::Mul => "*",
 					BinaryOperator::Div => "/",
+					BinaryOperator::FloorDiv => {
+						return format!(
+							"Math.floor({} / {})",
```

Comment:
to deal with negatives
```suggestion
							"Math.trunc({} / {})",
```

Reply:
TIL